test/cql-pytest/cassandra_tests/validation/operations/compact_storage_test.py

```diff
@@ -12,6 +12,12 @@
 
 compactOption = " WITH COMPACT STORAGE"
 
+# Enable compact_storage for all tests
+@pytest.fixture(scope="session")
```

Comment:
You are effectively enabling compact storage for the entire test session (for the entire invocation of `pytest` that is.
If you only want it enabled for this one file, use the `module` scope.

See https://docs.pytest.org/en/6.2.x/fixture.html#fixture-scopes.